### PR TITLE
Admin can log in 119 closes #119

### LINKED ISSUE #119: as an admin i can log in

### PR DESCRIPTION
**Pull request by mikedao**
Created a test to verify that when an admin user logs in, they were brought to the admin dashboard. This necessitated modifications to the Sessions Controller, and application controller.

Sessions controller now checks to see if user is an admin. If not, it will attempt authentication as a regular user.

Application controller handles @current_user differently now. If a user is an admin, the sessions hash now has a key :admin that is true. If it is a regular user, the value is nil. This will allow the application controller to successfully create a session for said user.

image

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -3,14 +3,27 @@ def new
   end
 
   def create
-    user = User.find_by(email: params[:session][:email])
-    if user && user.authenticate(params[:session][:password])
-      session[:user_id] = user.id
-      flash[:success] = "Successfully logged in!"
-      redirect_to(:back)
+    if Admin.find_by(email: params[:session][:email])
```
Comment (EmilyMB) [inline app/controllers/sessions_controller.rb:6]: Could this be broken out into one or more methods? It's a little hard to follow the logic here. :small_blue_diamond:

File: db/seeds.rb [hunk 2]
```diff
@@ -189,6 +190,12 @@ def generate_users
     ])
   end
 
+  def generate_admins
+    @admins = Admin.create(full_name: "Admin",
```
Comment (EmilyMB) [inline db/seeds.rb]: We need Josh Cheek here I think!!! (per specs)

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (mikedao) [inline app/controllers/sessions_controller.rb:6]: The methods, they have been broken out. SO BROKEN OUT.

image

Comment (EmilyMB) [inline app/controllers/sessions_controller.rb:6]: :100:
